src/FSLibrary/MissionSorobanInvokeHostLoad.fs

```diff
@@ -33,10 +34,26 @@ let sorobanInvokeHostLoad (context: MissionContext) =
         (fun (formation: StellarFormation) ->
             formation.WaitUntilSynced [ coreSet ]
             formation.UpgradeProtocolToLatest [ coreSet ]
-            formation.UpgradeMaxTxSetSize [ coreSet ] 100000
+
+            let maxTxSetSize = 10000000
+            formation.UpgradeMaxTxSetSize [ coreSet ] maxTxSetSize
 
             formation.RunLoadgen coreSet context.GenerateAccountCreationLoad
-            formation.UpgradeSorobanLedgerLimitsWithMultiplier [ coreSet ] 1000
+            formation.UpgradeSorobanLedgerLimitsWithMultiplier [ coreSet ] 3000
             formation.UpgradeSorobanTxLimitsWithMultiplier [ coreSet ] 100
-            formation.RunLoadgen coreSet context.SetupSorobanInvoke
-            formation.RunLoadgen coreSet context.GenerateSorobanInvokeLoad)
+
+            // Have a unique instance per TX, assume 6 seconds per ledger for wiggle room
+            let txsPerLedger = context.txRate * 6
+            let peer = formation.NetworkCfg.GetPeer coreSet 0
+
+            formation.RunLoadgen coreSet { context.SetupSorobanInvoke with instances = Some(txsPerLedger) }
+
+            formation.RunLoadgen
+                coreSet
+                { context.GenerateSorobanInvokeLoad with
+                      instances = Some(txsPerLedger)
+                      // Generate TXs up to max resources
+                      dataEntriesHigh = Some(peer.GetTxWriteEntries())
+                      ioKiloBytesHigh = Some(peer.GetTxWriteBytes() / 1024)
+                      txSizeBytesHigh = Some(maxTxSetSize / txsPerLedger)
```

Comment:
why do we derive byte size from transaction counts? probably want to use byte size per-tx limit

Reply:
In this test, the per-tx limits are so high such that we hit the maxTxSetSize first.

Reply:
what I'm wondering is why use tx counts to derive a byte value. Did you mean to use ledger_max_tx_size_bytes here? otherwise the `maxTxSetSize / txsPerLedger` divides tx counts